code/modules/mob/living/living.dm

```diff
@@ -1138,6 +1138,18 @@
 	mob_pickup(user)
 	return TRUE
 
+/mob/living/display_output(sound/S, image/I)
+	..()
+	if(I && audiolocation)
+		client.images += I
+		addtimer(CALLBACK(src, .proc/remove_image, I), 7)
+
+/mob/living/proc/remove_image(image)
+	if(image && client)
+		client.images -= image
+		qdel(image)
```

Comment:
This implies you have a seperate image for every single thing which hears the sound. Structure it so the image can be shared.

Reply:
You're still deleting the image